plugins/catalog/src/components/EntityLinksCard/EntityLinksCard.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Entity } from '@backstage/catalog-model';
+import { IconComponent, InfoCard, useApp } from '@backstage/core';
+import { useEntity } from '@backstage/plugin-catalog-react';
+import LanguageIcon from '@material-ui/icons/Language';
+import React from 'react';
+import { EntityLinksEmptyState } from './EntityLinksEmptyState';
+import { LinksGridList } from './LinksGridList';
+import { ColumnBreakpoints } from './types';
+
+type Props = {
+  /** @deprecated The entity is now grabbed from context instead */
+  entity?: Entity;
```

Comment:
I guess this can be removed?

Reply:
Was thinking that, but the EntityPage hasn't been updated so was trying to keep this consistent. I think it would only work if the integrator has updated the routing to use `FlatRoutes`. Fine either way here 😄

Reply:
Let's keep it around for now. It requires porting the app over to a flat EntityPage, using EntitySwitch etc.